package.json

```diff
@@ -18,20 +18,38 @@
   },
   "homepage": "https://github.com/prateekbh/babel-esm-plugin#readme",
   "dependencies": {
-    "ava": "^1.0.1",
-    "deepcopy": "^1.0.0",
-    "webpack": "^4.28.4"
+    "chalk": "^2.4.1",
+    "deepcopy": "^1.0.0"
   },
   "peerDependencies": {
     "webpack": "^4.28.4"
   },
   "devDependencies": {
     "@babel/core": "^7.2.2",
     "@babel/preset-env": "^7.2.3",
+    "ava": "^1.0.1",
     "babel-loader": "^8.0.5",
-    "chalk": "^2.4.1",
     "esm": "^3.0.84",
+    "husky": "^2.4.0",
+    "lint-staged": "^8.2.0",
+    "prettier": "^1.18.0",
     "release-it": "^12.0.1",
+    "webpack": "^4.28.4",
     "webpack-cli": "^3.2.1"
+  },
+  "lint-staged": {
+    "src/**/*.js": [
```

Comment:
you can use `{src,test}/**/*.js`

also `prettier --write` should be enough

Reply:
done, thanks :)

Reply:
@ForsakenHarmony i had to re-open these because tests have `fixtures/**/*.js` and i din't knwo a good way of excluding them in this single statement

Reply:
https://github.com/developit/microbundle/blob/master/lint-staged.config.js

Reply:
gotcha thanks :), u da best